### ISSUE TITLE
Whether or not use conan by remote in Clion

### ISSUE THREAD

**Issue by MichaelWU0726** [conan-io/conan-clion-plugin#100]
I followed [the official blog post](https://blog.jetbrains.com/clion/2019/05/getting-started-with-the-conan-clion-plugin/) about clion integration. But in menu `Build,Execution,Deployment > Conan`, I couldn't find to select the remote conan executable

**Comment by memsharded** [comment]
Hi @MichaelWU0726 

It is possible that there is something not working great with the plugin. The blog post is almost 3 years old, and recently we have stopped maintaining the plugins, because we are focused on Conan 2.0. We will resume work on it after 2.0.

Moving this issue to the CLion plugin repo.

**Comment by MichaelWU0726** [comment]
> Hi@MichaelWU0726
> 
> It is possible that there is something not working great with the plugin. The blog post is almost 3 years old, and recently we have stopped maintaining the plugins, because we are focused on Conan 2.0. We will resume work on it after 2.0.
> 
> Moving this issue to the CLion plugin repo.

when would the plugin work? Recently, I want to use conan in IDE

**Comment by memsharded** [comment]
> when would the plugin work? Recently, I want to use conan in IDE

It will still take a few months, I cannot say accurately, but 2.0 still in alpha, will move to beta for a few months before GA.

**Comment by MichaelWU0726** [comment]
> > when would the plugin work? Recently, I want to use conan in IDE
> 
> It will still take a few months, I cannot say accurately, but 2.0 still in alpha, will move to beta for a few months before GA.

So, it is only used by command line now?

**Comment by memsharded** [comment]
> So, it is only used by command line now?

Yes. You might want to try the ``cmake-conan`` integration, so CMake manages to retrieve the dependencies automatically (but be aware that that approach might also have some edges, like not supporting the latest Conan features).

In any case, the plugin barely implements like 5% of the whole functionality (``conan install``), so the command line is very recommended anyway, I strongly recommend to learn it and use it, it pays off.
